tls/s2n_cipher_suites.c

```diff
@@ -675,6 +736,20 @@ int s2n_cipher_suites_init(void)
                 break;
             }
         }
+
+        /* Initialize SSLv3 cipher suite if SSLv3 utilizes a different record algorithm */
+        if (cur_suite->sslv3_record_alg && cur_suite->sslv3_record_alg->cipher->is_available()) {
+            struct s2n_blob cur_suite_mem = {.data = (uint8_t *) cur_suite, .size = sizeof(struct s2n_cipher_suite)};
```

Comment:
spaces around the braces appears to be the convention.

Reply:
well, it looks like you are using convention, there is a little inconsistency just above this code change.